homeassistant/components/rainmachine.py

```diff
@@ -1,132 +0,0 @@
-"""
```

Comment:
Please use `git mv <X> <Y>` to keep history attached to a file

Reply:
Not sure what you mean here – I performed `git mv homeassistant/components/rainmachine.py homeassistant/components/rainmachine/__init__.py`, but the result appears to be the same as before